scripts/xtensa-build-zephyr.py

```diff
@@ -497,10 +497,12 @@ def build_platforms():
 		# Build
 		try:
 			execute_command(build_cmd, cwd=west_top)
-		except:
+		except subprocess.CalledProcessError as cpe:
 			zephyr_path = pathlib.Path(west_top, "zephyr")
 			if not os.path.exists(zephyr_path):
-				sys.exit("Zephyr project not found. Please run this script with -c flag or clone manually.")
+				sys.exit("Zephyr project not found. Please run this script with -u flag or `west update zephyr` manually.")
+			else: # unknown failure
+				raise cpe
```

Comment:
Why hide other exceptions. I prefer https://github.com/thesofproject/sof/pull/6097

Reply:
Other exceptions are not hidden, they're not caught and raised directly

Reply:
This fixes `pylint W0703: Catching too general exception Exception (broad-except)`

Reply:
Fair enough, the exception catching is filtered now as well, right.